radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/invariants/consensus/NodeEvents.java

```diff
@@ -39,6 +40,6 @@
 	}
 
 	public <T> EventProcessor<T> processor(BFTNode node, Class<T> eventClass) {
-		return t -> consumers.get(eventClass).forEach(c -> c.accept(node, t));
+		return t -> consumers.getOrDefault(eventClass, Collections.emptySet()).forEach(c -> c.accept(node, t));
```

Comment:
`Collections.emptySet()` can be replaced with `Set.of()` which is a little bit more concise.